src/createMollieClient.ts

```diff
@@ -46,7 +47,8 @@ function createHttpClient(options: MollieOptions): AxiosInstance {
   axiosOptions.headers['Accept-Encoding'] = 'gzip';
   axiosOptions.headers['Content-Type'] = 'application/json';
 
-  var customVersionStrings = options.versionStrings;
+  let customVersionStrings = options.versionStrings;
```

Comment:
let customVersionStrings = options.versionStrings || [];

will help the checks on line 52

Reply:
This, too, depends on how we want this function to handle edge-cases.

`options.versionStrings || [];` would change the behaviour of an empty string or the number 0 being passed as `versionStrings`.